smallrye-reactive-messaging-provider/src/main/java/io/smallrye/reactive/messaging/providers/helpers/PausableMulti.java

```diff
@@ -0,0 +1,107 @@
+package io.smallrye.reactive.messaging.providers.helpers;
+
+import java.util.concurrent.Flow;
+import java.util.concurrent.atomic.AtomicLong;
+import java.util.concurrent.locks.ReentrantLock;
+
+import io.smallrye.mutiny.Multi;
+import io.smallrye.mutiny.helpers.Subscriptions;
+import io.smallrye.mutiny.operators.MultiOperator;
+import io.smallrye.mutiny.operators.multi.MultiOperatorProcessor;
+import io.smallrye.mutiny.subscription.MultiSubscriber;
+import io.smallrye.reactive.messaging.PausableChannel;
+
+public class PausableMulti<T> extends MultiOperator<T, T> implements PausableChannel {
```

Comment:
Would like to have @jponge review on this one.

Reply:
I'll check that